gcp_audit_rules/gcp_iam_org_folder_changes.py

```diff
@@ -0,0 +1,28 @@
+from panther_base_helpers import deep_get
+
+def rule(event):
+    # Return True to match the log event and trigger an alert.
+    logname = deep_get(event, "logName")
+    return deep_get(event, "protoPayload", "methodName") == "SetIamPolicy" and \
+            (logname.startswith("organizations") or \
+            logname.startswith("folder") ) and \
+            logname.endswith("/logs/cloudaudit.googleapis.com%2Factivity") and \
+            deep_get(event,
+                "protoPayload",
+                "requestMetadata",
+                "callerSuppliedUserAgent").lower().find('terraform') == -1
```

Comment:
Just one thought, is there any value in still alerting in this case but using a custom severity function to set the severity to low or info? If you remove the last condition from here and added something like:

```
def severity(event):
  if "callerSuppliedUserAgent").lower().find('terraform') == -1:
    return 'INFO'
  return 'HIGH'
```

Then it would create an alert in either case, but for the terraform ones it would be at `INFO` severity.

Reply:
Accepted change recommendation for severity